### PR TITLE
testing my branch

### PR DESCRIPTION
**Pull request by samhachem99**

### CODE REVIEW

**Review by edwardpassagi — COMMENTED** [review]
do you mean "more stuff there" ?

File: index.html [hunk 1]
```diff
@@ -20,6 +20,7 @@
                 <h1>Welcome to the official site of</h1>
                 <h2>The Nutrition App</h2>
                 <h2>More Contents Are on The Way</h2>
+                <h2> More stuff here </h2>
```
Comment (edwardpassagi) [inline index.html:23]: do you mean "more stuff there" ?

Comment (samhachem99) [reply index.html:23]: no I meant here

Comment (edwardpassagi) [reply index.html:23]: ok, approved

**Pull request merged**